pkgs/tools/audio/tts/default.nix

```diff
@@ -0,0 +1,103 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, fetchpatch
+, python3
+}:
+
+#
+# Tested in the following setup:
+#
+# TTS model:
+#   Tacotron2 DDC
+#   https://drive.google.com/drive/folders/1Y_0PcB7W6apQChXtbt6v3fAiNwVf4ER5
+# Vocoder model:
+#   Multi-Band MelGAN
+#   https://drive.google.com/drive/folders/1XeRT0q4zm5gjERJqwmX5w84pMrD00cKD
+#
+# Start like this:
+#   tts-server \
+#     --vocoder_config /tmp/tts/vocoder/config.json \
+#     --vocoder_checkpoint /tmp/tts/vocoder/checkpoint_1450000.pth.tar \
+#     --tts_config /tmp/tts/config.json \
+#     --tts_checkpoint /tmp/tts/checkpoint_130000.pth.tar
```

Comment:
The default configuration contains also a `scale.py` which is read from the current directory. So `cd /tmp/tts` is required for this example to work.

Reply:
Expanded on the necessary directoy structure and the pwd.